lib/ddtrace/sampler.rb

```diff
@@ -247,7 +248,12 @@ def priority_assigned_upstream?(span)
 
     def priority_sample!(span)
       preserving_sampling(span) do
-        @priority_sampler.sample!(span)
+        @priority_sampler.sample!(span).tap do
+          # We want to make sure the span is tagged with the agent-derived
+          # service rate. Retrieve this from the rate by service sampler.
+          agent_rate = @priority_sampler.sample_rate(span)
```

Comment:
@brettlangdon This tag is meant to be "the rate limit returned by the agent." I assume this is the rate we used to sample? If so, how is this any different than `_sample_rate` when the priority sampler is used to sample?

Reply:
`_sample_rate` is for client sampling that tells the agent how to upscale metrics based on what is not sent to the agent.

since priority sampling always sends all traces to the agent we shouldn't set `_sample_rate`.